### PR TITLE
Mock Course Data

### LINKED ISSUE #15: Integrate OSU Course Data with platform data

### PR DESCRIPTION
**Pull request by cassiusfive**
## Changes
- Added model for courses
- Users now have an m-n relationship with courses
- Example courses are seeded

## Notes
We may want to support course specific tags in the feature. For example a user might want to tag a post with their specific lecture / recitation.

### CODE REVIEW

File: packages/database/prisma/schema.prisma [hunk 1]
```diff
@@ -18,6 +18,7 @@ model User {
   lastLogin  DateTime?
   Discussion Discussion[]
   Reply      Reply[]
+  courses    Course[]
```
Comment (Nyumat) [inline packages/database/prisma/schema.prisma]: nit: could we keep the casing?

**Review by Nyumat — CHANGES_REQUESTED** [review]
one small thing, but ur good after that

File: packages/database/prisma/schema.prisma (hunk 1 above)
Comment (cassiusfive) [reply packages/database/prisma/schema.prisma]: Resolved here -> 2a76572

**Review by Nyumat — APPROVED** [review]
Cheers 🚀

**Pull request merged**